packages/common-server/src/errorReporting.ts

```diff
@@ -25,6 +25,20 @@ export function rewriteFilename(filename: string) {
   }
 }
 
+// This offers some protection against accidentally sending too many errors to Sentry
+const BAD_ERROR_SAMPLE_RATE = 0.05;
```

Comment:
until we know where this is coming from and since we are almost at our quota, how about making this 0.001? (so 40 errors a day)

Reply:
Ok, changing it